homeassistant/components/schluter/climate.py

```diff
@@ -0,0 +1,171 @@
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, CONF_SCAN_INTERVAL, TEMP_CELSIUS
+
+from . import DATA_SCHLUTER, DOMAIN as SCHLUTER_DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=5)
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {vol.Optional(CONF_SCAN_INTERVAL): vol.All(vol.Coerce(int), vol.Range(min=1))}
+)
+
+MODE_SCHLUTER_TO_HASS = {
+    False: HVAC_MODE_OFF,
+    True: HVAC_MODE_HEAT,
+}
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Schluter thermostats."""
+    data = hass.data[DATA_SCHLUTER]
+    devices = []
+    temp_unit = hass.config.units.temperature_unit
+
+    for thermostat in data.thermostats:
+        devices.append(SchluterThermostat(thermostat, temp_unit, data))
+        _LOGGER.debug("Adding thermostat: %s", thermostat.name)
+
+    add_entities(devices, True)
+
+
+class SchluterThermostat(ClimateDevice):
+    """Representation of a Schluter thermostat."""
+
+    def __init__(self, device, temp_unit, data):
+        """Initialize the thermostat."""
+        self._unit = temp_unit
+        self.device = device
+        self.data = data
+
+        # Set the default supported features
+        self._support_flags = SUPPORT_TARGET_TEMPERATURE
+
+        # data attributes
+        self._serial_number = None
+        self._group = None
+        self._name = None
+        self._target_temperature = None
+        self._temperature = None
+        self._temperature_scale = None
+        self._is_heating = None
+        self._action = None
+        self._min_temperature = None
+        self._max_temperature = None
+
+    @property
+    def should_poll(self):
+        """Return if platform should poll."""
+        return True
+
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return self._support_flags
+
+    @property
+    def unique_id(self):
+        """Return unique ID for this device."""
+        return self.device.serial_number
+
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        return {
+            "identifiers": {(SCHLUTER_DOMAIN, self.device.serial_number)},
+            "name": self.device.name,
+            "manufacturer": "Schluter",
+            "model": "Thermostat",
+            "sw_version": self.device.sw_version,
+        }
+
+    @property
+    def name(self):
+        """Return the name of the thermostat."""
+        return self._name
+
+    @property
+    def temperature_unit(self):
+        """Return the unit of measurement."""
+        return self._unit
+
+    @property
+    def current_temperature(self):
+        """Return the current temperature."""
+        return self._temperature
+
+    @property
+    def hvac_mode(self):
+        """Return current operation ie. heat, idle."""
+        return MODE_SCHLUTER_TO_HASS[self._is_heating]
```

Comment:
Do we need a lookup for this?
```suggestion
        return HVAC_MODE_HEAT if self._is_heating else HVAC_MODE_HEAT
```

Reply:
Agreed, absolutely not. I've used your suggestion (with a slight change where the "else" is HVAC_MODE_OFF, and I've removed the no longer necessary MODE_SCHLUTER_TO_HASS